_summaries/srivastava2014dropout_1.md

```diff
@@ -1,19 +1,114 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: srivastava2014dropout
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: patelojas
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# Main Idea
+
+This paper looks to solve the problem of overfitting for large networks. It introduces dropout, a technique that randomly drops units and their connections from a neural network during training. 
+
+Current deep learning networks contain many layers, allowing them to learn complicated relationships from data. However, due to limitations in training data and the depth and complexity of modern data, some of the relationships are learned from overfitting and will not be seen in actual test data. 
+
+Ensemble methods work very well in many settings. These methods include combining several models, but is tough when considering large neural networks. Ensemble methods are the most effective when each individual model is unique from the rest, such as having a different architecture or being trained on different data. Given the current limits on computation power and smaller amounts of training data, having several unique networks is a difficult task. Instead, dropout simulates combining different neural networks together. 
+
+Dropout is a technique of randomly removing several nodes and their incoming and outgoing edges from the network. The removal probability of each node is independent from each other. We call the network post-removal as the "thinned" network. For each a training example, a new "thinned" network is trained, as the dropout is performed differently for every example. 
+
+One of the main motivations in the paper is adapting a network so that each unit can be trained to work with a random set of previous units as any unit can be randomly dropped. 
+
+![Diagram of Removal](srivastava2014dropout_1a.png)
+
+# Technical Implementation
+
+During testing and validation, we are unable to explicitly average the predictions from all thinned models. Instead, we use a single neural network without performing dropout, but scale the weights down with the dropout rate for each node. 
+
+![Diagram of Removal](srivastava2014dropout_1b.png)
+
+In terms of seeing a description of the model, we can see the following:
+
+![Diagram of Removal](srivastava2014dropout_1e.png)
+
+Here, we can treat $$y$$ as the output of a unit, $$z$$ as the input of a layer, $$f$$ as the activation function, $$w$$ as the weights, and $$b$$ as the biases. These values can be calculated using the following equations: 
+
+![Diagram of Removal](srivastava2014dropout_1c.png)
```

Comment:
Formula in latex might seem nicer.

Reply:
I'm having trouble typing latex in the md document.

Reply:
native markdown doesn't support latex but you can view the effect though `jekyll build`. But it's fine to use figure.